### ISSUE TITLE
Tooltip not showing on hover over icon

### ISSUE THREAD

**Issue by Skolo73** [Zren/plasma-applet-tiledmenu#100]
The same thing happens with Present windows button.
Thank You for your work!

**Comment by Zren** [comment]
As in the "This is the Tiled Menu widget" tooltop? I didn't bother adding any tooltip features since I don't need them. There's no point in adding a useless tooltip that'll appear when you fling the mouse to the bottom left corner to get it out of the way.

**Comment by Skolo73** [comment]
Yes, something simple like "Application launcher" or "Menu". Personally, I don't need it either, but I thought it would be more consistent with the other widgets in the panel.
